authors/settings.py

```diff
@@ -137,7 +137,7 @@
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
 
-    'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
-    ),
+    # 'DEFAULT_AUTHENTICATION_CLASSES': (
```

Comment:
@neelxie , why did you comment this part out. Also, please ensure that your `[PR naming](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices)` and `[commit message](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices)` conform to the team conventions.

Reply:
@cdvx let me work on this and fix it.